heroku_connect/utils.py

```diff
@@ -26,11 +27,12 @@ def get_mapping(version=1, exported_at=None):
     """
     if exported_at is None:
         exported_at = timezone.now()
+    app_name = app_name or settings.HEROKU_CONNECT_APP_NAME
```

Comment:
How about making `settings.HEROKU_CONNECT_APP_NAME` as default argument for `app_name`

Reply:
Again, I will add to the doc-string as it would make the function signature access the setting. Maybe the setting isn't set. The function should still work.